### ISSUE TITLE
message in supergroups

### ISSUE THREAD

**Issue by ticoti** [vysheng/tg#1067]
Since the new update of telegram for the 5k supergroups ( one week more or less) the messages of big supergroups are not arriving to telegram-cli but the messages in private or small groups, works perfectly

**Comment by adriandelarco** [comment]
I have the same problem

**Comment by richpeterson** [comment]
Me too

**Comment by davue** [comment]
seems like supergroups are not supported yet

**Comment by ticoti** [comment]
but in telegram-cli ? really strange because supergroups were working fine before

**Comment by adriandelarco** [comment]
@ticoti That's correct. Supergroups were working fine, since the last update to 5K members are not working fine in telegram-cli

**Comment by ticoti** [comment]
Now I have discovered that if I get the conversation open in one supergroup in telegram desktop, then the telegram-cli is receiving the messages in that supergroup

**Comment by dr-peterson** [comment]
how fixed this problem?

**Comment by toaskoas** [comment]
I have supergroups with 2, 20 or 30 members where its work. Only on two open supergroups with 9 and 32 members it doesnt work.

**Comment by Testato** [comment]
are you tried the actual Master version ?
i had the  same problem and today i recompiled the master and it work

**Comment by Testato** [comment]
Wait a moment, whit the last Master on the Group it work and on the Chat it do not work.
???

**Comment by ticoti** [comment]
I have tested the master version independently ( not with bot because it is not updated) and it doesn't work either

**Comment by Testato** [comment]
For "bot" you mean the lua parsing method ?
In this moment i have the actual Master version installed and the parsing on chat do not work, instead the parsing on supergroups work

**Comment by hard0ff** [comment]
Same problem

**Comment by BH-YAGHI** [comment]
Update lua-tg.c 


 tgl_do_channel_get_members (TLS, lua_ptr[p + 1].peer_id, 1000, 0, 0, lua_contact_list_cb, lua_ptr[p].ptr);

TO

  tgl_do_channel_get_members (TLS, lua_ptr[p + 1].peer_id, 5000, 0, 0, lua_contact_list_cb, lua_ptr[p].ptr);

**Comment by ticoti** [comment]
@BH-YAGHI is that solving the problem? it doesn't make sense to me because that is onyl related to the execution of that function.

Anyway, has to recompile it, sure?

**Comment by BH-YAGHI** [comment]
@ticoti https://github.com/SEEDTEAM/TeleSeed/commit/891b01291ad750a2d16270cd3716c2505be6ee93

**Comment by BH-YAGHI** [comment]
https://github.com/Rondoozle/tg/blob/ad44d65cd29b3aca58ee37439aa04a97d4cd59e7/lua-tg.c see this!

**Comment by ticoti** [comment]
But I tested Teleseed and it is not working properly either, I think the problem is in the own telegram-cli and not in the bots

**Comment by BH-YAGHI** [comment]
so why my bot is work ?!!!

**Comment by LeviDettwyler** [comment]
Same problem here. Sometimes a few messages show up, and they'll show up for a little while right after I do a channel_join, but stop shortly afterwards. Only way to read the messages in this supergroup is to use the history command.

@BH-YAGHI Tried this and unfortunately isn't fixing it for me.

**Comment by adel-s** [comment]
@BH-YAGHI thanks! This solution is worked for me.

**Comment by HellScre4m** [comment]
Long time passed. Still no solution?

**Comment by DubininaTaisiia** [comment]
I don't see any notifications in updates, when supergroup deleted.

**Comment by Jostino** [comment]
I've a friend with this issue. Any news about? only on TG web.

**Comment by Jeklah** [comment]
Nice @ghost that fixed it for me, thanks.
